website/common/script/content/gear/sets/armoire.js

```diff
@@ -5,6 +5,8 @@ let armor = {
   lunarArmor: {
     text: t('armorArmoireLunarArmorText'),
     notes: t('armorArmoireLunarArmorNotes', { str: 7, int: 7 }),
+    textlocaleKey: 'armorArmoireLunarArmorText',
```

Comment:
I don't think this is necessary, the server should return already translated texts